src/js/components/Menu/Menu.js

```diff
@@ -353,6 +354,7 @@ const Menu = forwardRef((props, ref) => {
               onDropClose();
             }
           }}
+          {...theme.menu.button}
```

Comment:
It seems like this should be above some of the other props in the button instead of at the end so that if we pass button props through `items` they don't get overridden by what is defined in the theme.

Reply:
I agree. If the caller used a specific style per item I would expect it to take precedence over a theme styling. Fixed.